lib/tictactoe-gui/ui/graphical_ui.rb

```diff
@@ -12,26 +12,30 @@ class GraphicalUi
       BOARD_TYPES_TEXT = 'Board Types'
       PLAY_BUTTON_TEXT = 'Play'
       PLAY_BUTTON_NAME = 'play_button'
+      PLAYER_TURN_TEXT = "Turn = Player %s"
 
       def initialize(parent)
         @parent = parent
         build_gui_objects
       end
 
-      def get_players_selection
-        @players_menu.selected_option
+      def draw(board, marker)
```

Comment:
Might be a fine name in most other cases, but there's another meaning for draw in TTT, might want to consider renaming to update or even render instead.